continuous-integration/dev-sec-ops/secret-management/secrets_rotation.md

```diff
@@ -0,0 +1,21 @@
+# Secrets Rotation
+
+Secret rotation is the process of refreshing the secrets that are used by the application.
+For further reading see [CSEDevSecOps Secret Management Scenarios](https://github.com/microsoft/CSEDevSecOps/tree/master/Scenarios/SecretManagement)
+
+## Why Secrets Rotation [The Why]
+
+Secrets are an asset and as such have a potential to be leaked or stolen. By rotating the secrets, we are revoking any secrets that may have been compromised. Therefore, secrets should be rotated frequently.
+
+## Applying Secrets Rotation [the how]
+
+To promote frequent rotation of a secret - define an automated periodic secret rotation process.

```

Comment:
It feels like we should have some reference to Managed Identities in this section that mentions how using a Managed Identity allows you to offload secret rotation to Azure, and should be our default choice as long as meets the security/compliance requirements of the customer.

I didn't see Managed Identities called out in the Secret Management Scenarios either, which makes it seem like its a secondary option to doing your own key rotation.

Reply:
Agree with @seushermsft

Reply:
@seushermsft you are totally right.  Added information about Managed Identities and that they should be our default choice.